### PR TITLE
fix: executor keep state 

### PR DESCRIPTION
**Pull request by samsja**

### CODE REVIEW

File: executor.py [hunk 1]
```diff
@@ -19,7 +21,7 @@ class SimpleIndexer(Executor):
     def __init__(
         self,
         match_args: Optional[Dict] = None,
-        table_name: Optional[str] = None,
```
Comment (cristianmtr) [inline executor.py:22]: not optional

**Review by cristianmtr — CHANGES_REQUESTED** [review]

File: executor.py [hunk 2]
```diff
@@ -19,7 +21,7 @@ class SimpleIndexer(Executor):
...
+        table_name: Optional[str] = TABLE_NAME,
```
Comment (nan-wang) [inline executor.py:24]:
```suggestion
        table_name: str = 'simple_indexer_table',
```

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: executor.py [hunk 3]
```diff
@@ -5,6 +5,8 @@
 from jina import DocumentArray, Executor, requests
 from jina.logging.logger import JinaLogger
 
+TABLE_NAME = 'simple_indexer_table'
+
```
Comment (nan-wang) [inline executor.py:9]:
```suggestion
```

**Review by nan-wang — CHANGES_REQUESTED** [review]

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**